ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -371,22 +383,29 @@ module Nvidia = struct
     in
     Pci.with_access (fun a -> build_vgpu_types a [] relevant_vgpu_confs)
 
-  let find_or_create_supported_types ~__context ~pci =
-    let dev_id = Xapi_pci.int_of_id (Db.PCI.get_device_id ~__context ~self:pci) in
-    let subsystem_dev_id =
-      match Db.PCI.get_subsystem_device_id ~__context ~self:pci with
-      | "" -> None
-      | id_string -> Some (Xapi_pci.int_of_id id_string)
-    in
-    debug "dev_id = %s" (Printf.sprintf "%04x" dev_id);
-    let relevant_types = relevant_vgpu_types dev_id subsystem_dev_id in
-    debug "Relevant vGPU configurations for pgpu = [ %s ]"
-      (String.concat "; "
-         (List.map (fun vt -> vt.model_name) relevant_types));
-    let vgpu_types = List.map
-        (fun v -> find_or_create ~__context v) relevant_types in
-    let passthrough_gpu_type = find_or_create ~__context passthrough_gpu in
-    passthrough_gpu_type :: vgpu_types
+  let find_or_create_supported_types ~__context ~pci
+        ~is_system_display_device
+        ~is_host_display_enabled
```

Comment:
Please would you add a comment saying that the last two parameters are unused, and are here only to match the function signature? Otherwise when reading this function in isolation it looks like a mistake.